src/plugins/discover/public/components/discover_grid_flyout/discover_grid_flyout.tsx

```diff
@@ -251,6 +254,15 @@ export function DiscoverGridFlyout({
             textBasedHits={isPlainRecord ? hits : undefined}
           />
         </EuiFlyoutBody>
+        <EuiFlyoutFooter>
```

Comment:
From the EUI docs (https://elastic.github.io/eui/#/layout/flyout#push-versus-overlay):

> Also, it is good to include a close button in the footer for a larger hit target than the small close button provides.